apps/deploy-web/src/utils/apiUtils.ts

```
@@ -111,7 +111,7 @@ export class ApiUrlService {
     return `${this.baseApiUrl}/v1/network-capacity`;
   }
   static gpuModels() {
-    return `${this.baseApiUrl}/internal/gpu-models`;
+    return `${this.baseApiUrl}/v1/gpu-models`;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)